bedrock/mozorg/templates/mozorg/about-en.html

```diff
@@ -0,0 +1,230 @@
+  {{ css_bundle('about-en') }}
+{% endblock %}
+
+{# TODO: migrate to global macro #}
+{% macro card_block(class, tag_label, media_icon, title, ga_title, desc, image_url, include_highres_image, video_url, aspect_ratio, link_url, meta) -%}
+<section class="mzp-c-card {% if class %}{{ class }}{% endif %} {% if aspect_ratio %}{{ aspect_ratio }}{% endif %} {% if media_icon %}{{ media_icon }}{% endif %} {% if video_url %}has-video-embed{% endif %}">
+  <a class="mzp-c-card-block-link" href="{{ link_url }}" data-link-name="{{ ga_title }}" data-link-type="link">
+    <div class="mzp-c-card-media-wrapper">
+      {{ lazy_img(image_url=image_url, include_highres_image=include_highres_image, placeholder_url='placeholder.png', optional_attributes={'class': 'mzp-c-card-image'}) }}
+    </div>
+    <div class="mzp-c-card-content">
+    {% if tag_label %}
+      <div class="mzp-c-card-tag {% if tag_class %}{{ tag_class }}{% endif %}">{{ tag_label }}</div>
+    {% endif %}
+      <h2 class="mzp-c-card-title"><span>{{ title }}</span></h2>
+    {% if desc %}
+      <p class="mzp-c-card-desc">{{ desc }}</p>
+    {% endif %}
+    {% if meta %}
+      <p class="mzp-c-card-meta">{{ meta }}</p>
+    {% endif %}
+    </div>
+  </a>
+  {% if video_url %}
+  <div class="mzp-c-card-video-wrapper hidden">
+    <figure class="mzp-c-card-video-content">
+      <video class="mzp-c-card-video" preload="none" controls playsinline data-ga-label="{{ ga_title }}">
+        <source src="{{ video_url }}.webm" type="video/webm">
+        <source src="{{ video_url }}.mp4" type="video/mp4">
+      </video>
+      <figcaption>
+        <p>{% if desc %}{{ desc }}{% endif %} <a href="{{ link_url }}">{{ _('Read more') }}</a></p>
+      </figcaption>
+    </figure>
+  </div>
+  {% endif %}
+</section>
+{%- endmacro %}
+
+
+{% block content %}
+<section class="mzp-c-hero">
+  <div class="mzp-l-content">
+    <div class="mzp-c-hero-body">
+      <h1 class="mzp-c-hero-title">{{ _('Mozilla makes browsers, apps, code and tools that put people before profit.') }}</h1>
+
+      <div class="mzp-c-hero-desc">
+        <p>{{ _('Our mission: Keep the internet open and accessible to all.') }}</p>
+      </div>
+
+      <p class="mzp-c-hero-cta">
+        <a class="mzp-c-button mzp-c-button-primary" href="{{ url('mozorg.about.manifesto') }}">{{ _('Read Our Mission') }}</a>
+      </p>
+    </div>
+  </div>
+
+</section>
+
+<div class="t-shade">
+  <div class="mzp-l-content">
+    <h1 class="c-page-title">{{ _('Our Mission in Action') }}</h1>
+    <div class="mzp-l-card-half">
+      {{ card_block(
+          class='mzp-c-card-medium',
+          title=_('Pioneers of The Open Web'),
+          ga_title='Pioneers of The Open Web',
+          desc=_('Our leadership has been at the forefront of building a healthier internet since Day 1. What began as an alternative to corporate domination has grown into a global force for good online.'),
+          image_url='mozorg/about/leaders.jpg',
```

Comment:
The low res images are being upsized in large viewports so they look blurry. The minimum size for a half-page card should probably be 600x400 but that means the retina images will be 1200x800 and that can make some really big files... maybe we can find a happy medium and just do a single size for all resolutions, like 800x600. Maybe that would look OK on retina?

Reply:
Some of the images are fine (Foundation, Careers) and some are blurry (container, IRL). Maybe it depends on the quality that was in the Sketch file I exported from? I... can't actually tell if the Leadership one is blurry because it's up-sized or blurry because it's taken in low light. I'll talk to Mark.

Reply:
Related: We should have guidelines on image dimensions in Protocol.

Reply:
Foundation and Careers are both being upscaled as well. They're 448px scaled up to 590px and 598px, respectively. I really only noticed this problem because my monitor is non-retina and the Firefox screenshot is unreadably blurry. The photos don't look quite as bad but they're still not as sharp as they would be if they weren't upscaled.

And now I'm looking at the current home page in production and it has some medium card images that are 465px upscaled to 598px. And looking in Protocol, the medium card component recommends a minimum width of 582px but even that would be scaled up a tiny bit to 598px. So... yeah, we really need to revise and document our image sizes in Protocol.

Reply:
Okay, I don't have versions of the assets big enough to generate the correct size high-res images. I'm going to remove the high res versions for now. Then I'll ask for what I need to make the high res versions.

Reply:
I think we're set with high res assets now, except the "Get Involved" image. That needs to be replaced. DNM until then.